src/Tomcat/TomcatController.ts

```diff
@@ -19,7 +19,13 @@ import { TomcatServer } from "./TomcatServer";
 import { WarPackage } from "./WarPackage";
 
 export class TomcatController {
+
+    private _javaExec: string ;
+
     constructor(private _tomcatModel: TomcatModel, private _extensionPath: string) {
+        // tslint:disable-next-line:no-backbone-get-set-outside-model
+        const javaHome: string = vscode.workspace.getConfiguration('java').get('home') ;
+        this._javaExec = `${javaHome}/bin/java` ;
```

Comment:
Maybe java.home as optional is better for current users. What if the configuration "java.home" not defined in vscode settings? It will break all our current users.

Reply:
Hello,
I made a modification => If java.home is not defined then the default behaviour is kept (java defined at system level through PATH system variable.
Thanks for your review